command/build.py

```diff
@@ -1,8 +1,37 @@
+import yaml as pyYaml
+import command.identify as identify
+
 class BuildCmd():
     name = 'build'
 
-    def __init__(self):	
+    def __init__(self):
         pass
+
+    def _get_empty_bundle(self):
+         return dict(
+            data = dict(
+                customResourceDefinitions = [],
+                clusterServiceVersions = [],
+                packages = [],
+            )
+        )
     
-    def build(self):
-        print("Hello, world. The build feature is currently under construction.")
+    def _get_field_entry(self, yamlContent):
+        yaml_type = identify.get_operator_artifact_type(yamlContent)
+        return yaml_type[0:1].lower() + yaml_type[1:] + 's'
+
+    def _updateBundle(self, operatorBundle, yamlContent):
+        operatorBundle["data"][self._get_field_entry(yamlContent)].append(pyYaml.load(yamlContent))
```

Comment:
Please always use `pyYaml.safe_load` to avoid security issues, unless you really need `yaml.load` and you can guarantee that `yamlContent` is safe.

Reply:
@MartinBasti addressed in latest commit.